scripts/Tools/mkDepends

```diff
@@ -346,7 +346,7 @@ foreach $f (keys %file_includes) {
     }
 
     # Expand
-    for ($i = 0; $i <= $#expand_incs; ++$i) {
```

Comment:
why is this change required?

Reply:
When running the model on Zeus with NEMO, the build of NEMO becomes slow and uses a lot of memory with the original code. 
With the suggested code this problem is solved.